guides/v2.1/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -13,6 +13,7 @@ The topic describes how to install, configure and use [Grunt JavaScript task run
 
 Make sure that you [set]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html) your Magento application to the developer or default [mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html).
 
+The default mode sets the Less compilation mode to Server-side Less compilation. 
```

Comment:
Could you actually move this to be a second sentence right after the sentence that begins on line 14 rather than its own line?